examples/kyb-app/src/common/utils/request.ts

```diff
@@ -2,6 +2,7 @@ import ky from 'ky';
 
 export const request = ky.extend({
   prefixUrl: import.meta.env.VITE_API_URL as string,
+  timeout: 10000,
```

Comment:
@chesterkmr it seems like the "/finish" request waits the all workflow to resolve his first run. it should just send the event and return. can you add a task to next the sprint?